src/Stratis.BreezeD/Program.cs

```diff
@@ -21,42 +21,38 @@ public static async Task Main(string[] args)
         {
             try
             {
-                // Get the API uri.
-                bool isTestNet = args.Contains("-testnet");
                 bool isStratis = args.Contains("stratis");
 
-                string agent = "Breeze";
-
                 NodeSettings nodeSettings;
 
                 if (isStratis)
                 {
-                    if (isTestNet)
-                        args = args.Append("-addnode=51.141.28.47").ToArray(); // TODO: fix this temp hack
-
-                    nodeSettings = new NodeSettings(networksSelector:Networks.Stratis, protocolVersion: ProtocolVersion.ALT_PROTOCOL_VERSION, agent: agent, args:args);
+                    nodeSettings = new NodeSettings(networksSelector: Networks.Stratis, protocolVersion: ProtocolVersion.PROVEN_HEADER_VERSION, agent: "Breeze", args: args)
+                    {
+                        MinProtocolVersion = ProtocolVersion.ALT_PROTOCOL_VERSION
+                    };
                 }
                 else
                 {
-                    nodeSettings = new NodeSettings(networksSelector:Networks.Bitcoin, agent: agent, args: args);
+                    nodeSettings = new NodeSettings(networksSelector: Networks.Bitcoin, agent: "Breeze", args: args);
                 }
 
                 IFullNodeBuilder fullNodeBuilder = new FullNodeBuilder()
                     .UseNodeSettings(nodeSettings)
+                    .UseBlockStore()
+                    .UsePosConsensus()
                     .UseLightWallet()
                     .UseBlockNotification()
                     .UseTransactionNotification()
                     .UseApi();
 
                 IFullNode node = fullNodeBuilder.Build();
 
-                // Start Full Node - this will also start the API.
-                if (node != null)
-                    await node.RunAsync();
+                await node.RunAsync();
```

Comment:
Why this change? It's exactly the same as StratisD.

Reply:
Ah I just thought its an unnecessary check as if the `node` was null here we actually might want to let startup break so that we are aware of it?